### ISSUE TITLE
No longer works

### ISSUE THREAD

**Issue by Baniita** [Dibakarroy1997/kisskh-dl#11]
```
C:\WINDOWS\system32>kisskh dl "https://kisskh.me/Drama/KinnPorsche-The-Series/Episode-12?id=4656&ep=95124&page=0&pageSize=100" -o "C:\Users\Bani\AppData\Local\Temp\KP"
Getting details for Episode 1...
[generic] ep.1.v0.1657615781.720: Downloading webpage
[generic] ep.1.v0.1657615781.720: Downloading m3u8 information
[info] ep.1.v0.1657615781.720: Downloading 1 format(s): 0
[hlsnative] Downloading m3u8 manifest
[hlsnative] Total fragments: 797
[download] Destination: C:\Users\Bani\AppData\Local\Temp\KP\Episode_12\Episode_12_E01.mp4
WARNING: The download speed shown is only of one thread. This is a known issue and patches are welcome
[download] fragment not found; Skipping fragment 1 ...
[download] fragment not found; Skipping fragment 2 ...
[download] fragment not found; Skipping fragment 3 ...
(goes on for a thousand fragments)
ERROR: The downloaded file is empty
Traceback (most recent call last):
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\Lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\Lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\Scripts\kisskh.exe\__main__.py", line 7, in <module>
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\kisskh_downloader\cli.py", line 71, in dl
    downloader.download_video_from_stream_url(video_stream_url, filepath, quality)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\kisskh_downloader\downloader.py", line 26, in download_video_from_stream_url
    ydl.download(video_stream_url)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\YoutubeDL.py", line 3322, in download
    self.__download_wrapper(self.extract_info)(
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\YoutubeDL.py", line 3297, in wrapper
    res = func(*args, **kwargs)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\YoutubeDL.py", line 1474, in extract_info
    return self.__extract_info(url, self.get_info_extractor(key), download, extra_info, process)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\YoutubeDL.py", line 1485, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\YoutubeDL.py", line 1582, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\YoutubeDL.py", line 1641, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\YoutubeDL.py", line 2737, in process_video_result
    self.process_info(new_info)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\YoutubeDL.py", line 3200, in process_info
    success, real_download = self.dl(temp_filename, info_dict)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\YoutubeDL.py", line 2924, in dl
    return fd.download(name, new_info, subtitle)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\downloader\common.py", line 446, in download
    ret = self.real_download(filename, info_dict)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\downloader\hls.py", line 354, in real_download
    return self.download_and_append_fragments(ctx, fragments, info_dict)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\downloader\fragment.py", line 532, in download_and_append_fragments
    return self._finish_frag_download(ctx, info_dict)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\downloader\fragment.py", line 307, in _finish_frag_download
    self.report_error('The downloaded file is empty')
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\YoutubeDL.py", line 1004, in report_error
    self.trouble(f'{self._format_err("ERROR:", self.Styles.ERROR)} {message}', *args, **kwargs)
  File "C:\Users\Bani\AppData\Local\Programs\Python\Python310\lib\site-packages\yt_dlp\YoutubeDL.py", line 944, in trouble
    raise DownloadError(message, exc_info)
yt_dlp.utils.DownloadError: ERROR: The downloaded file is empty
```

RIP?

**Comment by Dibakarroy1997** [comment]
Fix in latest release.

`pip install -U kisskh-downloader`

**Comment by Dibakarroy1997** [comment]
Fixed as part of 12385f81e7938b220fecc034454a83b557a18b66
